portal-impl/src/com/liferay/portal/model/impl/LayoutImpl.java

```diff
@@ -1351,7 +1351,7 @@ private LayoutTypePortlet _getLayoutTypePortletClone(
 	private List<PortletPreferences> _getPortletPreferences(long groupId) {
 		List<PortletPreferences> portletPreferences =
 			PortletPreferencesLocalServiceUtil.getPortletPreferences(
-				groupId, PortletKeys.PREFS_OWNER_TYPE_LAYOUT,
+				groupId, PortletKeys.PREFS_OWNER_TYPE_GROUP,
```

Comment:
Again, we're using the Group ID but looking for preferences of type Layout. It doesn't make sense to me, but perhaps you know better the reasoning behind this. It also causes issues with embedded portlets again since they are stored in the database using the Group ID and the Group type.